src/plugins/content/accessibility/jumplinks/plugin.js

```diff
@@ -50,40 +53,81 @@ export default pluginFactory({
     /**
      * Initializes the plugin (called during runner's init)
      */
-    init: function init() {
-        const mapJumpToAreaBroker = {
-            question: 'getContentArea',
-            navigation: 'getNavigationArea',
-            toolbox: 'getToolboxArea',
-            teststatus: 'getPanelArea',
-        };
-        this.jumplinks = jumplinksFactory({})
+    init() {
+        const testRunner = this.getTestRunner();
+        const item = testRunner.getCurrentItem();
+        const config = {
+            isReviewPanelEnabled: isReviewPanelEnabled(testRunner),
+            questionStatus: getItemStatus(item)
+        }
+
+        this.jumplinks = jumplinksFactory(config)
             .on('render', () => {
                 const closeShortcutsHandler = closeShortcuts.bind(this);
-                this.jumplinks.on('jump', (jump) => {
-                    const $element = this.getAreaBroker()[mapJumpToAreaBroker[jump]]().find(":not(.hidden)[tabindex]").first();
+                this.jumplinks.on('jump', (jumpTo) => {
+                    const areaBroker = this.getAreaBroker();
+                    const $element = getJumpElementFactory(areaBroker)[jumpTo];
                     $element.focus();
                 });
                 this.jumplinks.on('shortcuts', () => {
                     this.shortcuts.show();
                     this.shortcuts.getElement()
-                        .off("click", closeShortcutsHandler)
-                        .on("click", closeShortcutsHandler);
+                        .off('click', closeShortcutsHandler)
+                        .on('click', closeShortcutsHandler);
                     $(window)
-                        .off("keydown", closeShortcutsHandler)
-                        .on("keydown", closeShortcutsHandler);
+                        .off('keydown', closeShortcutsHandler)
+                        .on('keydown', closeShortcutsHandler);
                 });
+            })
+            .on('update', function update(params) {
+                this.trigger('changeReviewPanel', params.isReviewPanelEnabled);
+                this.trigger('changeQuesitionStatus', params.questionStatus);
+            })
+            .on('changeReviewPanel', function changeReviewPanel (enabled) {
+                const elem = this.getElement();
+                const panelJumplink = elem
+                    .find('[data-jump="teststatus"]')
+                    .parent();
+                if (enabled) {
+                    panelJumplink.removeClass('hidden');
+                } else {
+                    panelJumplink.addClass('hidden');
+                }
+            })
+            .on('changeQuesitionStatus', function changeQuesitionStatus(questionStatus) {
+                const elem = this.getElement();
+                const text = __('Question') + ' - ' + __(questionStatus);
```

Comment:
Translation has to be done upfront. We must not use variable or any other dynamic input for the `__()` helper.
Moreover, you should rather use string literals here to compose the string. Strange that your IDE didn't catch that.